### ISSUE TITLE
Number of sequences in /processed/pdb_labels  do not match the article and code

### ISSUE THREAD

**Issue by luoyu5023** [aim-uofa/VFN#5]
According to the traditional CATH4.2 dataset partitioning method and the train_multi_label.json and eval_multi_label.json in the code, theoretically the training set size is 18024 and the test set is 1120. However, the link you provided, https://zenodo.org/records/11369361, only downloads 18082. Is there any incomplete data?
